packages/tfjs-node-helpers/src/testing/confusion-matrix.ts

```diff
@@ -1,28 +1,24 @@
+import { math, Tensor } from '@tensorflow/tfjs-node';
+
 export class ConfusionMatrix {
   public tp: number;
   public tn: number;
   public fp: number;
   public fn: number;
 
-  constructor(trueValues: Float32Array, predictedValues: Float32Array) {
-    this.tp = 0;
-    this.tn = 0;
-    this.fp = 0;
-    this.fn = 0;
+  constructor(trueValues: Tensor, predictedValues: Tensor) {
+    const trueY = trueValues.as1D();
+    const predY = predictedValues.as1D();
+    const confusionMatrix = math.confusionMatrix(trueY, predY, 2);
+    const [[tn, fp], [fn, tp]] = confusionMatrix.arraySync();
 
-    for (let index = 0; index < trueValues.length; index++) {
-      const trueValue = trueValues[index];
-      const predictedValue = predictedValues[index];
+    trueY.dispose();
+    predY.dispose();
```

Comment:
@VladimirGlinskii It looks like `trueY` and `predY` could be disposed before calling `confusionMatrix.arraySync()`.